app/src/main/java/com/kickstarter/viewmodels/BackingFragmentViewModel.kt

```diff
@@ -534,13 +562,22 @@ interface BackingFragmentViewModel {
                         Backing.STATUS_COLLECTED -> R.string.We_collected_your_pledge_for_this_project
                         Backing.STATUS_DROPPED -> R.string.Your_pledge_was_dropped_because_of_payment_errors
                         Backing.STATUS_ERRORED -> R.string.We_cant_process_your_pledge_Please_update_your_payment_method
-                        Backing.STATUS_PLEDGED -> R.string.If_your_project_reaches_its_funding_goal_the_backer_will_be_charged_total_on_project_deadline
+                        Backing.STATUS_PLEDGED -> {
```

Comment:
@jlplks add tests for the new string, take a look here to see an example -> `@Test
    fun testFixPaymentMethodButtonIsGone_whenBackingIsErrored()` at BackingFragmentViewModelTest.kt